pilot/pkg/bootstrap/sidecarinjector.go

```diff
@@ -37,6 +37,8 @@ import (
 var (
 	injectionWebhookConfigName = env.RegisterStringVar("INJECTION_WEBHOOK_CONFIG_NAME", "istio-sidecar-injector",
 		"Name of the mutatingwebhookconfiguration to patch, if istioctl is not used.")
+	tlsCertFile = env.RegisterStringVar("TLS_SERVER_CERT_CHAIN", filepath.Join(dnsCertDir, "cert-chain.pem"), "File containing the x509 Certificate for HTTPS.")
```

Comment:
nit: `TLS_SERVER_CERT_CHAIN` seems too generic while the intention here is to use only for injector. Should we qualify it a bit more like `SIDECAR_INJECTOR_TLS_SERVER_CHAIN` etc?

Reply:
Added prefix `INJECTION_WEBHOOK_` to match the earlier env var.